engine/api/src/main/scala/pl/touk/nussknacker/engine/api/util/ReflectUtils.scala

```diff
@@ -1,7 +1,6 @@
-package pl.touk.nussknacker.engine.util
+package pl.touk.nussknacker.engine.api.util
```

Comment:
I wonder if we should move whole class to api? It's already bloated and we need just one method... Maybe it would be better to just split it?

Reply:
Indeed. Let's extract `StaticMethodRunner` as a separate class to utils

Reply:
Ok I see that you've moved also `ThreadUtils`. Let's split it then